satellite/views.py

```diff
@@ -509,6 +694,14 @@ def delete(request):
 
 
 def get_updated_views():
+    """
+    Create a dictionary with view name as key and the products under that key as value.
+    Display this dictionary in view existing tab of content view.
+
+    :param request: None
+
+    :returns:       Dictionary with all existing views with their product urls.
```

Comment:
instead of 'Dictionary with all existing views with their product urls' write 
"Dictionary with all existing views along with their respective product urls."